Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/border.h

```diff
@@ -198,8 +198,8 @@ namespace Polygon_mesh_processing {
     }
 
     //face index map given as a named parameter, or as an internal property map
-    FIMap fim = choose_param(get_param(np, CGAL::face_index),
-                             get_const_property_map(CGAL::face_index, pmesh));
+    FIMap fim = boost::choose_param(get_param(np, CGAL::face_index),
```

Comment:
As far as I understand, here we use non-documented function templates overloads from `<boost/graph/named_function_params.hpp>`.

1. Is not that dangerous? What is the plan is Boost internals are changed in an incompatible way?
2. Why do you need to qualify `choose_param`? It seems the argument-dependent lookup (ADL) should find it.
3. Why do you need to qualify `choose_param`, but not `get_param`?